### PR TITLE
text changes+ some minor reindentation around some messaging

### PR DESCRIPTION
**Pull request by zbtirrell**

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [tribe-options-addons-api.php:52](https://github.com/moderntribe/the-events-calendar/blob/25199addac42c54bc8b2d8f43a5f8492417b98e7/src/admin-views/tribe-options-addons-api.php#L52)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$time' | [tribe-options-addons-api.php:52](https://github.com/moderntribe/the-events-calendar/blob/25199addac42c54bc8b2d8f43a5f8492417b98e7/src/admin-views/tribe-options-addons-api.php#L52)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [tribe-options-addons-api.php:54](https://github.com/moderntribe/the-events-calendar/blob/25199addac42c54bc8b2d8f43a5f8492417b98e7/src/admin-views/tribe-options-addons-api.php#L54)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$time' | [tribe-options-addons-api.php:54](https://github.com/moderntribe/the-events-calendar/blob/25199addac42c54bc8b2d8f43a5f8492417b98e7/src/admin-views/tribe-options-addons-api.php#L54)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Tabs/New.php [hunk 1]
```diff
@@ -312,39 +312,40 @@ public function get_result_messages( $queue ) {
...
 					_n( '%1$d new event category was created.', '%1$d new event categories were created.', $queue->activity->count( 'category', 'created' ), 'the-events-calendar' ),
 					$queue->activity->count( 'category', 'created' )
 				) .
-									 ' <a href="' . admin_url( 'edit-tags.php?taxonomy=tribe_events_cat&post_type=tribe_events' ) . '">' .
-									 __( 'View your event categories', 'the-events-calendar' ) .
-									 '</a>';
-			;
+				' <a href="' . admin_url( 'edit-tags.php?taxonomy=tribe_events_cat&post_type=tribe_events' ) . '">' .
+				__( 'View your event categories', 'the-events-calendar' ) .
+				'</a>';
```
Comment (bordoni) [inline src/Tribe/Aggregator/Tabs/New.php]: Maybe these links we could do: 
```php
sprintf( // Build Category Link
	'<a href="%s">%s</a>',
	admin_url( 'edit-tags.php?taxonomy=tribe_events_cat&post_type=tribe_events' ),
	esc_html__( 'View your event categories', 'the-events-calendar' )
);
```

File: src/Tribe/Aggregator/Tabs/New.php [hunk 2]
```diff
@@ -312,39 +312,40 @@ public function get_result_messages( $queue ) {
...
-											 ' <a href="' . admin_url( 'edit.php?post_type=tribe_organizer' ) . '">' .
-											 __( 'View your event organizers', 'the-events-calendar' ) .
-											 '</a>';
-					;
+					$messages['success'][] = '<br/>' .
+					sprintf(
+						_n( '%1$d new organizer was imported.', '%1$d new organizers were imported.', $queue->activity->count( 'organizer', 'created' ), 'the-events-calendar' ),
+						$queue->activity->count( 'organizer', 'created' )
+					) .
+					' <a href="' . admin_url( 'edit.php?post_type=tribe_organizer' ) . '">' .
```
Comment (bordoni) [inline src/Tribe/Aggregator/Tabs/New.php]: Same thing for the link here.

**Comment by zbtirrell** [comment]
comments added to all the `sprintf`'s. Plus a minor refactor to remove one that wasn't needed.

**Pull request merged**